site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,128 @@ For a go app the manifest snippet would look like this:
+
+1. Creating the lambda functions with appropriate policies.
+2. Storing static assets in the dedicated S3 bucket.
+3. Storing the app’s manifest file in the same dedicated S3 bucket.
+
+#### Mattermost setup
+
+Follow the Prerequisites section of the [JS](../quick-start-js) or [GO](../quick-start-go) Quick Start Guides to set up your Mattermost instance. Ensure that you install the [apps plugin](https://developers.mattermost.com/integrate/apps/quick-start-js/#install-the-apps-plugin) as part of this process.
+
+#### AWS setup
+
+##### 1. Create AWS S3 bucket
+
+You will need to create an S3 bucket within AWS or use an existing bucket.
+
+1. Visit https://s3.console.aws.amazon.com
+1. Create a bucket.
+1. **Bucket name**: Give your bucket a name.
+1. **AWS Region**:
+    1. Select your region
+    1. Save the slug value for later (Example: `us-east-1`)
+    1. Corresponding Lambdas will be provisioned in the same region
+1. **Block Public Access settings for this bucket**
+    1. Uncheck **Block *all* public access**
+    1. Check **Block public access to buckets and objects granted through new public bucket or access point policies**
+    1. Check **Block public and cross-account access to buckets and objects through any public bucket or access point policies**
+    1. Check **I acknowledge that the current settings might result in this bucket and the objects within becoming public.**
+1. Select **Create Bucket**.
+
+##### 2. Create an IAM user access key and secret
+
+You will need an access key and secret so that `appsctl` can provision the app. These credentials can come from creating an IAM user, using a privileged IAM user, or even using the AWS account owners personal access key. Please follow the instructions [provided by AWS](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/) to complete these steps and save the `Access key ID` and `Secret access key` values.
+
+##### 3. Set AWS Provision environment variables
+
+Open a terminal where you installed the Apps plugin and set the following variables to the AWS credentials just created and saved:
+
+- `MM_APPS_PROVISION_AWS_ACCESS_KEY`  
+- `MM_APPS_PROVISION_AWS_SECRET_KEY`
+
+Set the following environment variables based on bucket name and region
+
+- `MM_APPS_S3_BUCKET`  
+- `MM_APPS_AWS_REGION`
+
+##### 4. Initialize the AWS resources
 
-### Deploying third-party apps to AWS
+The following command will create a Mattermost invoke user and security policy for use with AWS.
 
-Note that third-party apps are not supported in the Mattermost Cloud - they're used only for developer testing convenience. Provisioning in the third-party AWS cloud environment is done by the **appsctl** tool using the command:
+`go run ./cmd/appsctl aws init --create --create-access-key`
```

Comment:
What was the reason to move away from the `go install` way?

Reply:
@DHaussermann and I ran into an issue where he had installed appsctl before, but it was an older version of the code.  We could still use the installed version, but need to ensure it is updated.

Reply:
The nice thing about `go install` is that it (by default) install the latest tagged version not `master` making it more stable.